packages/client/src/hooks/useTreasuryTransactions.js

```diff
@@ -6,34 +6,17 @@ const NETWORK = process.env.REACT_APP_FLOW_ENV === "testnet" ? "test" : "main";
 const API_URL = `https://prod-${NETWORK}-net-dashboard-api.azurewebsites.net/api`;
 const FETCH_URL = `${API_URL}/company/${COMPANY_ID}/search?projectId=${PROJECT_ID}`;
```

Comment:
I still don't think it's a good idea to have configs hardcoded in the source code, I'll work on implementing a solution for the configs https://linear.app/dappercollectives/issue/VSL-214

Reply:
we could make `REACT_APP_GRAFFLE_API_URL_TESTNET` and `REACT_APP_GRAFFLE_API_URL_MAINNET` and then switch which one we use based on `NETWORK` but its just more code and this shouldn't need to change / be configured that often

Reply:
I'm thinking more like a `useConfig` hook, to hide all these implementation details of string concatenation